.goreleaser.yaml

```diff
@@ -0,0 +1,41 @@
+# Make sure to check the documentation at https://goreleaser.com
```

Comment:
Is this still needed? I don't see anything using it.

Reply:
It came from the boilerplate. I'll modify it because I like to have pointers to docs for third-party components, even when the link is obvious